hackathon_site/event/jinja2/event/landing.html

```diff
@@ -27,292 +29,296 @@
-            <div class="col s12 m6">
-                <ul class="collapsible">
-                    <li>
-                        <div class="collapsible-header">
-                            What is a hackathon?
-                        </div>
-                        <div class="collapsible-body">
-                            <span>A hackathon is a big bonanza where people from everyone doing everything come together to create projects together with the newest and most exciting tech.</span>
-                        </div>
-                    </li>
-                    <li>
-                        <div class="collapsible-header">
-                            How much does it cost?
-                        </div>
-                        <div class="collapsible-body">
-                            <span>$0! IEEE UofT is a non-profit organization that aims to provide students accessible technical events.</span>
-                        </div>
-                    </li>
-                    <li>
-                        <div class="collapsible-header">
-                            When can I expect to hear back after I applied?
-                        </div>
-                        <div class="collapsible-body">
-                            <span>
-                                We are having 2 rounds of acceptance. If you applied early, you might get an email on the weekend of Oct 10th/11th. If you still haven’t gotten any replies by then,
-                                don’t worry! We will be sending out a second round of acceptance on the weekend of Oct 24th/25th.
-                            </span>
-                        </div>
-                    </li>
-                </ul>
-            </div>
+    <br /><br />
+    <div class="container">
+        <div class="section scrollspy" id="faq">
+            <div class="row">
+                <div class="col s12 m12 center">
+                    <h3 class="center white-text">FAQ</h3>
+                </div>
+                <div class="col s12 m6">
+                    <ul class="collapsible">
+                        <li>
+                            <div class="collapsible-header">
+                                What is a hackathon?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>A hackathon is a big bonanza where people from everyone doing everything come together to create projects together with the newest and most exciting tech.</span>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="collapsible-header">
+                                How much does it cost?
+                            </div>
+                            <div class="collapsible-body">
+                                <span>$0! IEEE UofT is a non-profit organization that aims to provide students accessible technical events.</span>
```

Comment:
Are we actually a non-profit? Can we give charitable recepits for sponsorships? Just wondering about the legal language